File: java/tests/CalculatorTest.java
package xavi_montero.dressing_grapple.java.tests;

import org.junit.Test;
import static org.junit.Assert.assertEquals;
import xavi_montero.dressing_grapple.java.Calculator;

public class CalculatorTest
{
    @Test
    public void testSum()
    {
        int a = 3;
        int b = 4;
        Calculator calculator = new Calculator();

        assertEquals( a + b, calculator.sum( a, b ) );
    }
}

File: php/tests/CalculatorTest.php
<?php

namespace XaviMontero\DressingGrapple\Php\Tests;

require_once( "http://localhost:8080/JavaBridge/java/Java.inc" );

class CalculatorTest extends \PHPUnit_Framework_TestCase
{
    public function testCreationIsOfProperClass()
    {
        $sut = new \Java( "xavi_montero.dressing_grapple.java.Calculator" );

        $this->assertInstanceOf( 'Java', $sut );
    }
}
